Sources/muterCore/CLICommands/RunCommand/Steps/DiscoverSourceFiles.swift

```diff
@@ -1,6 +1,5 @@
 import Foundation
 import Pathos
-import Curry
```

Comment:
Does this mean we can drop Curry as a dependency?

Reply:
good catch, I can't say for sure..need to double check
as far as I know it was using to facilitate the usage of `Commandant`